include/librealsense2/dds/dds-server.cmake

```diff
@@ -13,7 +13,7 @@ set(DDS_SERVER_FILES
 )
 
 source_group( "DDS" FILES ${DDS_SERVER_FILES} )
-
+include("${REPO_ROOT}/include/librealsense2/dds/topics/dds-topics.cmake")
```

Comment:
Can we put these at the end, like the other cmake files? (after target_sources)

Reply:
👍

Reply:
Done